model/src/main/proto/analytics.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+// Supported event actions for analytics logging
+enum EventAction{
+  // The undefined user action of an event
+  UNKNOWN_EVENT_ACTION = 0;
+}
+
+// Supported priority of events for analytics logging
+enum Priority{
+  // The undefined priority of an event
+  UNKNOWN_PRIORITY = 0;
```

Comment:
Can be rename this to `UNKNOWN`. `_PRIORITY` as a suffix has not been used for other two enums so just thinking about the consistency here.

Reply:
Renamed.

Reply:
Okay. Sounds good.